README.adoc

```diff
@@ -14,9 +14,9 @@ permalink: /guides/app-logging/
 :repo-description: Visit the https://kabanero.io/guides/{projectid}.html[website] for the rendered version of the guide.
 = Application Logging on OKD with Elasticsearch, FluentD, and Kibana
 
-In a Kubernetes environment, logs are constantly produced by the running pod processes. In order to effectively manage all the log data generated by a Kubernetes system, especially to prevent the loss of log data when a pod terminates, a log aggregation tool should be deployed on the user's OKD cluster to take control of all the logging activities. Modern log aggregating tools allow users to persist, search, and visualize their log data gathered from all the pods across their OKD clusters. There are a few log aggregating products available on the market that offer similar capabilities. This guide will walk through the process of deploying such a log aggregating tool called EFK stack using the ansible-playbook script provided by the OKD community.
+A Kubernetes environment constantly produce logs by its running pod processes. To effectively manage this log data and ensure no loss of log data occurs when a pod terminates, deploy a a log aggregation tool on the user's OKD cluster to control the logging activities. Log aggregation tools help users persist, search, and visualize the log data that is gathered from the pods across the OKD cluster. There are several log aggregation tools in the market that offer similar capabilities. This guide describes the process of deploying a log aggregating tool, **EFK (Elasticsearch, FluentD, and Kibana)**, using the ansible-playbook script provided by the OKD.
```

Comment:
17) --->change phrase --deploy **a a** log aggregation tool-- TO --deploy a log aggregation tool--

Last sentence ---- change --This guide describes the process of deploying a log aggregating tool -- TO -- This guide describes the process of deploying a log **aggregation** tool --

Reply:
@fwji @donbourne Don, I approve your change, but we still have that sentence that says..... deploy a a log aggregation tool on the user's OKD cluster......  (need to drop one "a"...)

Reply:
1. removed extra 'a'
2. 'aggregating' to 'aggregation'